Lib/test/test_cppext.py

```diff
@@ -72,16 +81,9 @@ def run_cmd(operation, cmd):
                     self.fail(
                         f"{operation} failed with exit code {proc.returncode}")
 
-        # Build the C++ extension
-        cmd = [python, '-X', 'dev',
-               SETUP_TESTCPPEXT, 'build_ext', '--verbose']
-        if std_cpp03:
-            cmd.append('-std=c++03')
-        run_cmd('Build', cmd)
-
-        # Install the C++ extension
+        # Build and install the C++ extension
         cmd = [python, '-X', 'dev',
-               SETUP_TESTCPPEXT, 'install']
+               '-m', 'pip', 'install', PKG_CPPEXTDATA]
```

Comment:
LGTM on the docs now, thanks.

You might be already on it, but seems like this is failing on [some of the buildbots](https://buildbot.python.org/all/#/builders/465/builds/1049) with

```
subprocess.CalledProcessError: Command '['env/bin/python', '-X', 'dev', '-m', 'pip', 'install', '/buildbot/buildarea/cpython/pull_request.ware-gentoo-x86.installed/build/target/lib/python3.12/test/cppextdata']' returned non-zero exit status 1.
```

And reporting the following error:

```
ERROR: Invalid requirement: '/buildbot/buildarea/cpython/pull_request.ware-gentoo-x86.installed/build/target/lib/python3.12/test/cppextdata'
```

Reply:
I'm aware -- these specific buildbots have been consistently failing since the start, hence me doing triggering runs on them with smaller changes to the test setup. :)

I've not been able to reproduce the error so far outside of those environments, and not had the time to investigate what's going on with these buildbots.

Reply:
Okay, thanks. I've dug in a bit and found the following (which you probably already did, but just documenting this for completeness for any others reading this):

* s390x RHEL7 & 8 refleaks failures with `test_asyncio` appears to be pre-existing pre-existing and error identically on other older and newer PRs
* s390x Fedora LTO + PGO failure with missing `make` target likewise fails the same way with older and newer PRs, so evidently also unrelated
* AMD64 Windows11 Bigmem test error with `test_hashlib` does not appear to be exactly consistent with others, some of which passed and some failed with different errors, but it has very little build history to go on, it appears rather flaky (it is bigmem after all) and the test failure here is only on a single run and doesn't seem obviously related, so maybe we can defer it for now.
* The remaining failures all occur if and only if the buildbot is an `Installed` buildbot (all Linux, various distros and platforms), where `pip` apparently cannot find the `cppextdata` directory with the C extension to build at the installed path. Given the search path, `__file__ / "cppextdata"` is different than for the non-installed versions (under `build/target/lib/python3.12/test/cppextdata` rather than `build/Lib/test/cppextdata`, I'm assuming the `cppextdata` is not present in that specific directory when installed, and is instead somewhere else or not installed/copied.